csrc/scheduler/utils.cpp

```diff
@@ -781,6 +781,48 @@ getScopePersistenceFactors(
 
 } // namespace
 
+int64_t getOnePersistentBufferSize(
```

Comment:
Is this just factored out of `persistentBufferSize` with no change?

Reply:
yes.